netcore3.1/global.json

```diff
@@ -0,0 +1,6 @@
+{
+    "sdk": {
+        "version": "3.1.x",
```

Comment:
It looks like the .x wildcard isn't accepted. A specific version needs to be specified as the minimum allowed. https://docs.microsoft.com/en-us/dotnet/core/tools/global-json?tabs=netcore3x#examples